### PR TITLE
MSR: Add cosmetic option to change volume of music and ambience

### PR DESCRIPTION
**Pull request by dyceron**
image

Requires #7117 to be merged first

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid: Samus Returns
 
+- Added: Changing the volume of the music and background ambience is now possible via cosmetic options.
```
Comment (Miepee) [inline CHANGELOG.md:38]: is background ambience = generic sfx? or is it some other background music thing?

File: test/games/samus_returns/gui/dialog/test_msr_cosmetic_patches_dialog.py [hunk 2]
```diff
@@ -46,6 +46,28 @@ def test_change_music_option(
...
+    dialog = MSRCosmeticPatchesDialog(None, cosmetic_patches)
+    label = getattr(dialog, label_name)
+    skip_qtbot.addWidget(dialog)
+    assert label.text() == "  0%"
+
+    slider = getattr(dialog, slider_name)
+    slider.setValue(80)
+
+    assert dialog.cosmetic_patches == MSRCosmeticPatches(**{field_name: 80})  # type: ignore[arg-type]
+    assert label.text() == " 80%"
```
Comment (Miepee) [inline test/games/samus_returns/gui/dialog/test_msr_cosmetic_patches_dialog.py:68]: assert that the underlying cosmetic field changes too please

File: CHANGELOG.md (hunk 1 above)
Comment (dyceron) [reply CHANGELOG.md:38]: Yeah it's the sfx for rooms. Don't have regular sfx yet

File: test/games/samus_returns/gui/dialog/test_msr_cosmetic_patches_dialog.py (hunk 2 above)
Comment (dyceron) [reply test/games/samus_returns/gui/dialog/test_msr_cosmetic_patches_dialog.py:68]: I don't know what you mean

Comment (Miepee) [reply test/games/samus_returns/gui/dialog/test_msr_cosmetic_patches_dialog.py:68]: You only assert that the text field of the slider changes
but we want to make sure that when the slider is changed, that the underlying value (e.g. dialog.cosmetic_patches.ambience_volume)  also changes.

Comment (Miepee) [reply test/games/samus_returns/gui/dialog/test_msr_cosmetic_patches_dialog.py:68]: dont assert the slider value, assert the underlying field

Comment (Miepee) [reply test/games/samus_returns/gui/dialog/test_msr_cosmetic_patches_dialog.py:68]: nvm, didnt see that you were already doing that, whoops.

**Review by Miepee — APPROVED** [review]

**Pull request merged**